jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/BuildImageMojoIntegrationTest.java

```diff
@@ -75,6 +72,21 @@ private static String buildAndRun(Path projectRoot, String imageReference)
 
   @Test
   public void testExecute_simple() throws VerificationException, IOException, InterruptedException {
+    // Test empty output error
+    try {
+      Verifier verifier = new Verifier(simpleTestProject.getProjectRoot().toString());
+      verifier.setAutoclean(false);
+      verifier.executeGoals(Arrays.asList("clean", "jib:build"));
+      Assert.fail();
+    } catch (VerificationException ex) {
+      Assert.assertTrue(
```

Comment:
This can be the same assertThat with CoreMatchers.contains

Reply:
I wanted to avoid the extra import, but I guess it doesn't matter that much.